consensus/polybft/fsm.go

```diff
@@ -289,6 +322,15 @@ func (f *fsm) createCommitEpochTx() (*types.Transaction, error) {
 	return createStateTransactionWithData(contracts.EpochManagerContract, input), nil
 }
 
+func (f *fsm) createValidatorSetCommit() (*types.Transaction, error) {
```

Comment:
I do not like this. Why would you create a local field called `commitValidatorSetInput` and then set it in a different function, and then use it here, when you can just create the commit validator set transaction straight away if the delta is not empty. Please remove this function.